src/index.js

```diff
@@ -31,9 +31,9 @@ var App = React.createClass({
 render((
   <Router history={browserHistory}>
     <Route path='/' component={App}>
-      <Route path='moti' component={MotiController} />
-      <Route path='climo' component={AppController} />
-      <Route path='compare' component={DualController} />
+      <Route path='/moti' component={MotiController} />
+      <Route path='/climo/:ensemble_name' component={AppController} />
+      <Route path='/compare/:ensemble_name' component={DualController} />
```

Comment:
Nice

Reply:
But should you be [using `Router` components, not `Route`](https://stackoverflow.com/a/41555097/1858846) here?

Reply:
Huh? I *am* using the Router component...

Reply:
Sorry, I must have misread the code.

Reply:
I will continue researching this.

Reply:
Yeah, if you look at the page errors, you'll see:

```
Loading failed for the <script> with source “http://localhost:8080/climo/bundle.js”.
```

So when you do an initial page load at one of the child paths, it tries to load all of the resources in the index.html (e.g. bundle.js (which is why you see nothing), all of the leaflet stuff, etc.) *from that path* and fails. But if you do the initial page load at the root, it gets all of those resources, and then never has to reload the page on navigation, because React takes care of all of the navigation through re-rendering components.

But yeah, if you have any suggestions, I'd appreciate them.